src/openrct2/core/DataSerialiser.h

```diff
@@ -17,18 +17,21 @@ class DataSerialiser
 {
 private:
     MemoryStream _stream;
-    MemoryStream* _activeStream;
+    IStream* _activeStream;
     bool _isSaving;
+    bool _isLogging;
```

Comment:
this should have an initialiser here

Reply:
Its already set in the constructor initialization list.